website/docs/installation/documentdb/kubernetes.md

```diff
@@ -0,0 +1,94 @@
+---
+sidebar_position: 3
+---
+
+# Kubernetes
+
+FerretDB uses PostgreSQL with [DocumentDB extension](https://github.com/microsoft/documentdb) as the database engine.
+
+Ensure to have a running Kubernetes cluster before proceeding with the installation.
+
+You can deploy PostgreSQL with DocumentDB extension using any of our provided images.
+Please see the [Docker installation docs](../documentdb/docker.md) to learn more about the available images.
+
+:::tip
+We strongly recommend specifying the full image tag (e.g., `17-0.103.0-ferretdb-2.2.0`)
+to ensure consistency across deployments.
+For more information on the best FerretDB image to use, see the [DocumentDB release notes](https://github.com/FerretDB/documentdb/releases/).
+:::
+
+Create a `postgres.yaml` manifest with the following content:
```

Comment:
That's not a manifest, actually – that's two manifests. And you also can't create a manifest.

That's just a file in this case.

Reply:
In this situation, it is still called a manifest, not two manifests.

Here's a link to the Kubernetes docs where this is referenced and called a "manifest" despite two resources being deployed.
https://kubernetes.io/docs/concepts/workloads/management/

It's explained here as well as being the file itself:
https://kubernetes.io/docs/concepts/overview/working-with-objects/#:~:text=Most%20often%2C%20you%20provide%20the%20information%20to%20kubectl%20in%20a%20file%20known%20as%20a%20manifest.

Reply:
https://kubernetes.io/docs/reference/glossary/?all=true#term-manifest

> Specification of a Kubernetes API object in [JSON](https://www.json.org/json-en.html) or [YAML](https://yaml.org/) format.
> A manifest specifies the desired state of an object that Kubernetes will maintain when you apply the manifest. For YAML format, each file can contain multiple manifests.

Reply:
I'll update this to use file for this use case. I'm not quite sure of the use of "manifests" for a single file.